docs/release-notes/3735-agent-instance-stats-track.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
```

Comment:
Please provide more context about the problem being solved, as described in the Jira issue **Description**. But generalize it, omitting customer names and specific details.
See https://determinedai.atlassian.net/l/c/0MgNvgJT for some ideas about release note documentation.